test/layout/led_test.exs

```diff
@@ -0,0 +1,10 @@
+defmodule LayoutLEDTest do
+  use ExUnit.Case
+
+  alias Layout.LED
+
+  test "new/3 takes 3 arguments and creates a %Layout.LED{} struct" do
```

Comment:
My two cents: The name of this test is overly-specific and can be understood from reading the body of the test. The behavior that we care about testing can be simplified to something like `test "new creates a layout"`. This turns the tests into another form of documentation so that another developer can read the test descriptions and understand how the module should behave, rather than how it's implemented (because if they're refactoring, they may change the implementation).

Reply:
Yep, that sounds like a good idea. :+1: